Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN apt-get update && \
         curl \
         file \
         git \
+        golang \
```

Comment:
Interesting!

I have two questions about this:

1. Do you know how much size the golang toolchain adds to the base image?
2. It looks like the `grpcio` crate has ~32,000 downloads on crates.io. This is relatively large, and maybe enough to justifying including the `golang` toolchain if it's small enough. But are there other high-profile packages which need `golang` to build? (And can they build with the `golang` shipped by Ubuntu?)

The alternative here is that people could add `golang` to their own project's `Dockerfile`.

Reply:
Hm, I'm unsure about (1). Any idea as to how best I should measure that?

(2) I'd imagine very few other than the crates that require grpcio, or at least I've never encountered one.

Reply:
I did some digging around for (1), and it looks like the Ubuntu `golang` packages might pull in about 140 MB (mostly for the [compiler](https://packages.ubuntu.com/bionic/golang-1.10-go), assuming I'm reading things correctly). This is big enough to make a real difference to users of the image.

However, it's also really easy for users to add `golang` if they need it. Just make a `Dockerfile` [similar to this one](https://github.com/emk/rust-musl-builder/blob/master/examples/linking-with-git2/Dockerfile), and add the package you need using `RUN` (ideally before adding the source code to the image, for faster builds).

So I'm definitely interested in the `g++` link, but I'm leaning towards leaving out `golang` for now, because it's large, only needed by `grpcio`, and very easy to add when needed. What do you think?

I would, however, happily accept an example (in the `examples` directory) of how to use `grpcio`, and a note in the README linking to it.

Reply:
I think that makes sense - though, without golang I don't think any crates that rely on grpcio will work? So it feels like maybe the right move is to just have a separate dockerfile in /examples/ with the grpcio link *and* the golang dependency?

Reply:
Ah, you answered my question - use a new Dockerfile with the golang dep added.